### ISSUE TITLE
Support for multi-process mode in Basilisk/Serpent?

### ISSUE THREAD

**Issue by RamonUnch** [JustOff/github-wc-polyfill#33]
First I would like to thanks you for your work.
I like using the multi-process mode of Basilisk. to do this you can set
`browser.tabs.remote.force-enable ; true`
in about:config options.
However it seems The extension stops working correctly if this option is enabled, ie github homepage no longer display activity, post menu no longer fills etc.
I understand that multi-process mode is still experimental in Basilisk, but this plugin would be more useful if it supported this mode. I could not find old issue about that. So if you think it is irrelevant , just close it.

To reproduce, just download the latest build of Basilisk from roytam1 and in a new clean profile, add the `browser.tabs.remote.force-enable=true` boolean option in the `about:config` panel. Install your plugin and try it, it will fail unless you disable the previous option.

**Comment by JustOff** [comment]
Unfortunately, this extension cannot work with e10s enabled because Mozilla refused to fix [bug #1261585](https://bugzilla.mozilla.org/show_bug.cgi?id=1261585).

**Comment by andika207** [comment]
@RamonUnch I am unable to get multi process function to work.

@Vangelis66 so the multi-process doesn't currently run with Basilisk ?

Serpent and original Firefox 52 browsers work just fine though 
https://fossbytes.com/how-to-make-firefox-faster-by-enabling-multi-process-e10s-manually/

**Comment by RamonUnch** [comment]
In theory in Basilisk the e10s were disabled and cannot really be enabled, even if the options are still there and some people are able to do it (at least for some versions)
https://forum.palemoon.org/viewtopic.php?f=61&t=27786

On Serpent e10s were not removed.

**Comment by andika207** [comment]
@RamonUnch it's not in theory, it's for real. 
this is a nearly 1 year old thread so I can't be certain multi process was enabled back then because I simply had not updated Basilisk since 2020 until an update message recently emerged

anyway with some many alternatives for W7 I don't need this crap unsuble browser which struggels because of the lack of multi-process and doesn't even support Firefox addons

I checked the link out and it's funny, the same clown devs claiming this feature is dangerous or doesn't bring anything useful.
maybe they are clever than everyone else because all other browsers do support multi processes for a good reason....

**Comment by andika207** [comment]
> To reproduce, just download the latest build of Basilisk from roytam1

@RamonUnch after re-reading the first post I realized that you were talking about Serpent so this thread is pure BS 
you have no clue or don't check your words before posting thus this got me into the confusion

**Comment by Vangelis66** [comment]
> because I simply had not updated Basilisk since 2020 
**until an update message recently emerged**

At some point, **Moonchild** stopped producing **32-bit** binaries for (official) **Basilisk** browser; you didn't specify which architecture variant you had back in 2020 😉 , but, FWIW, the **last x86** build released by **MCP** was v`52.9.2020.11.25` ... 

**MCP** did continue to release **64-bit only** builds until v`52.9.2021.12.13`, which was the last one to contain an internal updater... A last, **EoS**, build **without an internal updater** was released (by **MCP**) in late January, v`52.9.2022.01.27` ...

In [late July](https://forum.palemoon.org/viewtopic.php?f=61&t=28657&sid=08d2c0ccff50a316fdda0a2dc96f0670), the (trademarked) **Basilisk** application (an **abandonware** during Feb to July 2022 😞 ) was sold and transferred to a new maintainer, **Basilisk-Dev**, who issued a first, long overdue, release in the start of Aug, v`52.9.2022.08.06`, now again with an internal updater built-in (plus several other, profile-breaking, changes, that you are advised to read about [here](https://www.basilisk-browser.org/releasenotes.shtml)) ...

> you have no clue or don't check your words before posting 
thus this got me into the **confusion**

...  Avoiding such eventual "confusions" was exactly what was addressed by my [previous post above](https://github.com/JustOff/github-wc-polyfill/issues/33#issuecomment-937250402) 😉 ...

> so the multi-process doesn't currently run with Basilisk ?

In [Basilisk features](https://www.basilisk-browser.org/features.shtml) it is plainly stated that:  
```
Important differences with Mozilla Firefox:
(...)
Does not use Electrolysis (e10s, multi-process browsing).
```
**MCP** were quick to excise e10s supporting code from the platform (**UXP**), and that was/is tracked in https://repo.palemoon.org/MoonchildProductions/UXP/issues/953; as you can see, e10s code in **Basilisk** has been removed...
I examined myself latest v`52.2022.08.06` release and I could locate only two vestigial prefs inside `about:config`: 
```
browser.tabs.remote.autostart
browser.tabs.remote.desktopbehavior
```
that do **not work** towards enabling e10s in **Bk**; same with the `browser.tabs.remote.force-enable` hidden pref... So, as you already discovered, e10s **can't be force-enabled** in **latest** Basilisk...

The Pale Moon forum thread linked by **Ramon** was dated **Jan 11th 2022**, so _**if**_ forced e10s did work, it would have to involve **previous Bk** builds (and the starter of that thread neglected to mention the exact build he experimented with...).
